MediaGalleryUi/view/adminhtml/web/js/directory/directoryTree.js

```diff
@@ -96,10 +104,17 @@ define([
          * Verify directory filter on init event, select folder per directory filter state
          */
         checkChipFiltersState: function () {
-            if (!_.isUndefined(this.filterChips().filters.path) && this.filterChips().filters.path !== '') {
+            var currentFilterPath = this.filterChips().filters.path,
+                currentTreePath = Base64.decode(window.MediabrowserUtility.pathId.replace(/--|,,/, ''));
+
+            if (!_.isUndefined(currentFilterPath) && currentFilterPath !== '' &&
```

Comment:
I would extract this if statement to a function, just to make it easier to read and make the intended explicit.